src/rpcrawtransaction.cpp

```diff
@@ -88,8 +94,17 @@ Array TxJoinSplitToJSON(const CTransaction& tx) {
             joinsplit.push_back(Pair("macs", macs));
         }
 
-        joinsplit.push_back(Pair("vpub_old", ValueFromAmount(jsdescription.vpub_old)));
-        joinsplit.push_back(Pair("vpub_new", ValueFromAmount(jsdescription.vpub_new)));
+        CDataStream ssProof(SER_NETWORK, PROTOCOL_VERSION);
```

Comment:
We need to check that this renders data in the expected order. The numbers and `uint256` fields in this method are reversed from what appears in the hex of a block, but the non-`uint256` fields are reversed. I believe this is consistent with Bitcoin because the non-`uint256` fields are vectors or arrays in the backend, but this was the root of my original uncertainty that led to me not including these at launch.